identity-service/latest/upgrade/index.md

```diff
@@ -4,20 +4,23 @@ title: Upgrade Identity Service
 
 Use the following information to upgrade the Identity Service to version 2.0.
 
-> **Note:** After the upgrade the database will no longer be compatible with the old server.
+> **Important:**
+>
+> * Upgrading the Identity Service requires downtime and should be performed in a test environment before being attempted in a production environment.
+> * After the upgrade the database will no longer be compatible with the old server.
 
-> **Important:** Upgrading the Identity Service requires downtime and should be performed in a test environment before being attempted in a production environment.
+Before performing an upgrade, make sure you review the recommended guidelines in the following sections:
 
 * [Upgrade from version 1.2](#upgrade-v12)
 * [Remove SmallRye references](#remove-smallrye-references)
 * [Upgrade ZIP installation](#upgrade-zip-installation)  
-* [Upgrade Kubernetes deployment with PostgreSQL database](#upgrade-kubernetes-deployment-with-postgresql-database)  
+  * [Upgrade Kubernetes deployment with PostgreSQL database](#upgrade-kubernetes-deployment-with-postgresql-database)  
```

Comment:
Are these 2 blank spaces wanted?

Reply:
Thanks for spotting that. I think it's incorrect that it's a subheading to the ZIP installation part, as the Kubernetes deployment does not rely on the ZIP distribution, but on the helm charts instead.

I'd treat it as a separate section instead (moving it 1 level up as you said).